### PR TITLE
Remove qeb-hwt and github-app bits

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Related-to: https://github.com/thoth-station/user-api/issues/1454

### CODE REVIEW

File: thoth/common/openshift.py [hunk 1]
```diff
@@ -1308,22 +1292,9 @@ def verify_kebechet_inputs(
     def verify_integration_inputs(
         self,
         source_type: Optional[ThothAdviserIntegrationEnum],
-        github_event_type: Optional[str] = None,
-        github_check_run_id: Optional[int] = None,
-        github_installation_id: Optional[int] = None,
-        github_base_repo_url: Optional[str] = None,
```
Comment (KPostOffice) [inline thoth/common/openshift.py:1314]: These need to be removed from schedule_adviser below I believe

Comment (pacospace) [reply thoth/common/openshift.py:1314]: Thanks!

**Review by harshad16 — COMMENTED** [review]
/lgtm
thanks :100:

**Review by KPostOffice — APPROVED** [review]

**Pull request merged**